### PR TITLE
Adding daily budget section for short blaze campaigns

### PR DESCRIPTION
**Pull request by sretika**
See 1242-ghe-Tumblr/a8c-dsp

## Proposed Changes

* For short (less than a week) Blaze campaigns, we should show daily budget instead of weekly budget.

#### Before
image

#### After
image

## Why are these changes being made?
* A user has reported that they are confused by seeing a "$35.00 weekly budget" for a 5 day campaign with $5 budget per day because the campaign doesn't run for a week.

## Testing Instructions

* Visit a (< 1 week) campaign detail page from `/advertising/campaigns/`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx [hunk 1]
```diff
@@ -646,22 +649,37 @@ export default function CampaignItemDetails( props: Props ) {
...
-													args: { dailyAverageSpending: dailyAverageSpending },
-												} )
-											) : (
-												<FlexibleSkeleton />
-											) }
-										</span>
-									</div>
+									{ isLessThanOneWeek ? (
+										<div>
+											<span className="campaign-item-details__label">{ __( 'Daily budget' ) }</span>
```
Comment (therocket-gr) [inline client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: i think we need a isLoading condition here and display the skeleton. 
as the page loads (I have a weekly campaign) I see the Daily budget when the page loads.. then it switches to weekly

Comment (therocket-gr) [reply client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: https://github.com/user-attachments/assets/101f147d-0cfb-4130-9af1-ff1d27505378

File: client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx [hunk 2]
```diff
@@ -224,6 +225,8 @@ export default function CampaignItemDetails( props: Props ) {
 		  )
 		: '';
 
+	const isLessThanOneWeek = false; //! is_evergreen && activeDays < 7;
```
Comment (sbarbosa) [inline client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: Don't forget to revert this hardcoded value back to `! is_evergreen && activeDays < 7`

**Review by sbarbosa — APPROVED** [review]
The code looks good and works as described. 
I left a comment about a hardcoded value that you may still need to change.

File: client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx (hunk 2 above)
Comment (therocket-gr) [reply client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: holy.. thanks for noticing this.. yes i hardcoded this to test it out.. nice catch. fixed it @sbarbosa

**Review by jjolmo — APPROVED** [review]
LGTM, nice addition!

**Review by j6ll — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16700404

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @sretika for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16700404).

**Pull request merged**